zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/store/JWTStore.java

```diff
@@ -0,0 +1,98 @@
+package com.github.onsdigital.zebedee.session.store;
+
+import java.util.HashMap;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import com.github.onsdigital.zebedee.user.model.User;
+import com.github.onsdigital.zebedee.session.model.Session;
+import com.github.onsdigital.zebedee.session.service.Sessions;
+import com.github.onsdigital.session.service.error.SessionClientException;
+
+/**
+ * JWTStore:  class, when instantiated, will
+ * *********  allow for access token verification,
+ *            storage (set()) and retrieval (get())
+ *            from threadlocal.
+ *           
+ *            Implements Sessions interface.
+ */
+public class JWTStore implements Sessions {
+
+    public HashMap<String, String> rsaKeyMap;
+
+    // class constructor - takes HashMap<String, String> as param.
+    public JWTStore(HashMap<String, String> rsaKeyMap) {
```

Comment:
Can you update this to `Map<String, String>` please? (Map is the interface type that HashMap implements but it gives us a bit more flexibility).